oddbird/oddcontrast
# move socials to footer, add icons

![](https://source.unsplash.com/featured/?cute,animal&CbsbxcnmvbE)


## Description
Add social landing page icons and links, and move this to the footer (was in header)


## Steps to test/reproduce
There are three linked icons in the footer now. Test to see if each icon shows up, and if each link goes to the correct place.

- [x] Do we want to show Twitter still? OddSite does. 
- [x] Do we want to show Mastodon moving forward?

## Design
https://xd.adobe.com/view/10ffb4e8-f459-422f-8e44-eb0701489393-45ac/screen/b6a20ffb-1389-42f4-9d44-4415a319d2af/specs/


## Show me
![oc-footer-medium](https://user-images.githubusercontent.com/1581694/226995493-0c784e4a-bc4b-4d33-b69f-de1f7738d968.jpg)
![oc-footer-large](https://user-images.githubusercontent.com/1581694/226995497-f6a99a85-0307-4265-b4c2-cffa5f1d76a1.jpg)
![oc-footer-small](https://user-images.githubusercontent.com/1581694/226995499-5e95e974-a8a5-4aa8-88c6-acb88d743ec7.jpg)

## Trello
https://trello.com/c/SXpuSkSm/45-i-want-social-icons
